sympy/assumptions/refine.py

```diff
@@ -332,6 +334,22 @@ def refine_sign(expr, assumptions):
     return expr
 
 
+def refine_matrixelement(expr, assumptions):
+    """
+    Examples
+    ========
+    >>>refine(X[0, 1], Q.symmetric(X))
```

Comment:
imports?

Reply:
In the updated commit I have added the imports.